src/test/scala/sangria/schema/ArgsSpec.scala

```diff
@@ -0,0 +1,158 @@
+package sangria.schema
+
+import org.scalatest.{Matchers, WordSpec}
+import sangria.execution.AttributeCoercionError
+
+import scala.collection.concurrent.TrieMap
+import sangria.marshalling.sprayJson._
+import spray.json._
+
+class ArgsSpec extends WordSpec with Matchers {
+
+
+  val NON_DEFAULT_ARGUMENT_NAME = "nonDefaultArgument"
```

Comment:
I would suggest to use [standard scala naming convention for constants](http://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods). For example `NonDefaultArgumentName` instead of `NON_DEFAULT_ARGUMENT_NAME `